.github/workflows/abi_bindings_checker.yml

```diff
@@ -33,7 +33,24 @@ jobs:
           export GOPATH=$HOME/go
           export PATH="$PATH:$BASE_DIR/.foundry/bin"
           export PATH="$PATH:$GOPATH/bin"
-          ./scripts/abi_go_bindings.sh
+          ./scripts/abi_bindings.sh
 
       - name: Check for clean branch
         run: .github/workflows/check_clean_branch.sh
+  unit_tests:
+    name: Unit tests
+    runs-on: ubuntu-20.04
+
+    steps:
+      - name: Setup Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Checkout repositories and submodules
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+
+      - name: Run ABI Binding Unit Tests
```

Comment:
nit: these are not just abi binding unit tests right?

Reply:
Good catch. Added a `cd abi-bindings/go` so it will only run the ABI bindings unit tests. Running `go test ./...` will call the E2E tests. They won't run since the necessary env var isn't set, but that's still not what we're targeting here.